src/PrestaShopBundle/Resources/config/services/bundle/form/form_type.yml

```diff
@@ -1581,6 +1580,7 @@ services:
 
     form.type.sell.product.data_transformer.typeahead_redirection_target_transformer:
         class: 'PrestaShopBundle\Form\Admin\Sell\Product\DataTransformer\TypeaheadRedirectionTargetTransformer'
+        deprecated: 'The "%service_id%" service is deprecated since 1.7.9 and will be removed in next major.'
         public: true
```

Comment:
Thanks @jolelievre. I use it for explain it in devdocs. https://github.com/PrestaShop/docs/pull/1034

:hearts: